src/main/java/teammates/ui/controller/InstructorCommentsPageData.java

```diff
@@ -126,4 +195,95 @@ public boolean isInstructorAllowedForPrivilegeOnComment(CommentAttributes commen
             return false;
         }
     }
+
+    public void init(Boolean isViewingDraft, Boolean isDisplayArchive, String courseId,
+                                    String courseName, List<String> coursePaginationList,
+                                    Map<String, List<CommentAttributes>> comments,
+                                    String string, InstructorAttributes instructor, CourseRoster roster,
```

Comment:
Is this `string` supposed to be the instructor's email?
It's possible to just get it from the `instructor` variable you're also passing in instead of keeping it as a separate variable.